4.classify-single-cell-phenotypes/4b.derive-classification-profiles/4b.derive-classification-profiles.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<p>May consider adding a markdown objective description at the top</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/cell-health-data/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I think that this is not necessary here given the info in the readme file.